docs/lib/datastore/fragments/ios/getting-started/40_codegen.md

```diff
@@ -1,14 +1,16 @@
-If you have chosen to use the Amplify/Tools pod, we will now use it to generate the models from the schema defined in `schema.graphql` and automatically add the new files to the project. If you have chosen to use the Amplify CLI instead, please skip this section, and follow the [Code generation: Amplify CLI](#code-generation-amplify-cli) section.
+On iOS the aforementioned `amplify-app` offers an Xcode integration that automatically adds the Amplify-specific files to your project.
 
-1. **Update the amplifytools.xcconfig file** and change `modelgen=false` to:
+1. **Run the command**:
     ```
-    modelgen=true
+    amplify codegen models && amplify-app
```

Comment:
I guess this is the reason, why you are asking to do a global install above? Might be good to just cut out amplify-app and go directly to the full amplify-cli workflow. You can still create project templates by running: `amplify init --quickstart`. then you could have this step be: `amplify codegen models && npx amplify-app`

Reply:
Ideally, we'd want to move the code for that adds the files to the Xcode project into the `amplify codegen models` workflow.

Reply:
I would avoid `npx amplify-app` due to reasons stated in my previous response above. I'm ok with changing the former to `amplify init --quickstart`.

ps: why does `amplify-app` exist as a separate package instead of a sub-command of the CLI?